docs/src/modules/components/Demo.js

```diff
@@ -271,12 +271,17 @@ const DemoRootJoy = joyStyled('div', {
   }),
 }));
 
-const DemoCodeViewer = styled(HighlightedCode)({
+const DemoCodeViewer = styled(HighlightedCode)(({ theme }) => ({
   '& pre': {
-    margin: '0 auto',
+    margin: 0,
     maxHeight: 'min(68vh, 1000px)',
+    maxWidth: 'initial',
+    borderRadius: 0,
+    [theme.breakpoints.up('sm')]: {
+      borderRadius: theme.shape.borderRadius,
+    },
   },
-});
+}));
```

Comment:
Fix demos that have `, "disableLiveEdit":true}}` on mobile, Before:

<img width="418" alt="Screenshot 2022-11-12 at 17 28 46" src="https://user-images.githubusercontent.com/3165635/201484206-16934aa4-1e09-4eeb-954e-555d0ee664e9.png">

After:

<img width="402" alt="Screenshot 2022-11-12 at 17 29 20" src="https://user-images.githubusercontent.com/3165635/201484227-c940dbc5-17e9-4904-b2fc-61f4db624367.png">